src/network/connectionserviceprovider.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CONNECTIONSERVICEPROVIDER_H
+#define CONNECTIONSERVICEPROVIDER_H
+
+#include <QObject>
+#include "localconnector.h"
+
+class ConnectionServiceProvider : public QObject
+{
+    Q_OBJECT
+
+    std::shared_ptr<DefaultLocalConnector> m_apiParser;
+
+    std::string m_userName;
```

Comment:
Code style should be the same 

std::string m_userName;
mContent

Reply:
Sorry :((
I forgot to remove these outdated files from repository. QT doesn't show files that are not included in the build process
![image](https://user-images.githubusercontent.com/18701555/62309125-8408c100-b48f-11e9-9610-77a8ce0bdeb9.png)